### PR TITLE
post onboarding navigation bar implementation

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

**Comment by DhaaraniCIT** [comment]
This will act as master for postonboarding

**Comment by DhaaraniCIT** [comment]
https://user-images.githubusercontent.com/55541808/218455412-77e140af-0698-4959-bbda-945322128941.mov

File: src/app/integrations/qbd/main/main.component.ts [hunk 1]
```diff
@@ -7,9 +8,18 @@ import { Component, OnInit } from '@angular/core';
 })
 export class MainComponent implements OnInit {
 
+  items: MenuItem[] = [
```
Comment (ashwin1111) [inline src/app/integrations/qbd/main/main.component.ts:11]: can we name it `modules` please?

File: src/app/integrations/qbd/main/main.component.ts [hunk 2]
```diff
@@ -7,9 +8,18 @@ import { Component, OnInit } from '@angular/core';
...
+    {label: 'Dashboard', routerLink: '/integrations/qbd/main/dashboard'},
+    {label: 'Configuration', routerLink: '/integrations/qbd/main/configuration'}
+  ];
+
+  activeItem: MenuItem;
```
Comment (ashwin1111) [inline src/app/integrations/qbd/main/main.component.ts:16]: activeModule or activeMenu please

File: src/app/integrations/qbd/main/main.module.ts [hunk 3]
```diff
@@ -2,14 +2,15 @@ import { NgModule } from '@angular/core';
...
-
+// Import { TabMenuModule } from 'primeng/tabmenu';
 
 @NgModule({
   declarations: [],
   imports: [
     CommonModule,
     MainRoutingModule,
     SharedModule
+    // TabMenuModule,
```
Comment (ashwin1111) [inline src/app/integrations/qbd/main/main.module.ts:13]: remove unwanted code

**Review by ashwin1111 — APPROVED** [review]

File: src/app/integrations/qbd/main/main.component.ts (hunk 1 above)
Comment (DhaaraniCIT) [reply src/app/integrations/qbd/main/main.component.ts:11]: it is a inbuild module of menu in primeng

Comment (ashwin1111) [reply src/app/integrations/qbd/main/main.component.ts:11]: I am suggesting for the variable name and not the type

**Pull request merged**